guides/keras_cv/object_detection_keras_cv.py

```diff
@@ -586,139 +607,60 @@ def dict_to_tuple(inputs):
 """
 ### Loss functions
 
-You may not be familiar with the `"focal"` or `"smoothl1"` losses.  While not
-common in other models, these losses are more or less staples in the object
+You may not be familiar with the `"ciou"` loss.  While not
+common in other models, this loss is more or less staple in the object
 detection world.
 
-In short, ["Focal Loss"](https://arxiv.org/abs/1708.02002) places extra emphasis
-on difficult training examples.  This is useful when training the classification
-loss, as the majority of the losses are assigned to the background class.
-
-"SmoothL1 Loss" is used to [prevent exploding gradients](https://arxiv.org/abs/1504.08083)
-that often occur when attempting to perform the box regression task.
+In short, ["Complete IoU"](https://arxiv.org/abs/1911.08287) is a flavour of the Intersection over Union loss and is used due to its convergence properties.
 
-In KerasCV you can use these losses simply by passing the strings `"focal"` and
-`"smoothl1"` to `compile()`:
+In KerasCV, you can use this loss simply by passing the string `"ciou"` to `compile()`.
+We also use standard binary crossentropy loss for the class head.
 """
 
 pretrained_model.compile(
-    classification_loss="focal",
-    box_loss="smoothl1",
+    classification_loss="binary_crossentropy",
+    box_loss="ciou",
 )
 
 """
 ### Metric evaluation
 
-Just like any other metric, you can pass the `KerasCV` object detection metrics
-to `compile()`.  The most popular object detection metrics are COCO metrics,
+The most popular object detection metrics are COCO metrics,
 which were published alongside the MSCOCO dataset. KerasCV provides an
-easy-to-use suite of COCO metrics under the `keras_cv.metrics.BoxCOCOMetrics`
-symbol:
+easy-to-use suite of COCO metrics under the `keras_cv.callbacks.PyCOCOCallback`
+symbol. Note that we use a Keras callback instead of a Keras metric to compute
+COCO metrics. This is because computing COCO metrics requires storing all of a
+model's predictions for the entire evaluation dataset in memory at once, which
+is impractical to do during training time.
 """
 
-coco_metrics = keras_cv.metrics.BoxCOCOMetrics(
-    bounding_box_format="xywh", evaluate_freq=20
+coco_metrics_callback = keras_cv.callbacks.PyCOCOCallback(
```

Comment:
It looks like we're no longer evaluating the pretrained model using the COCO metrics. The original goal here was to show the user some realistic values for these without having to train the model e2e. Is this something we could add back?

Reply:
We are still doing this, because when we show how to train the model we're using the PyCOCOCallback during training and we're training an already-pretrained model, so we can see the numerical results from the callback in that block.

IMO this whole block was very weird because it added a bunch of code that doesn't exist in KerasCV which really distracts from our API

Reply:
Whoops -- this is actually not true. I forgot that I had switched to using a pre-trained backbone without a pre-trained model head.

If you'd like I can add a block about how to just evaluate the metrics, or I can just add a text blurb about expected values. Wdyt @jbischof

Reply:
Yes, it should be relatively low friction to add some metrics to the completely pretrained model, I hope!

Reply:
yep -- added.